src/plugins/demo-app/index.tsx

```diff
@@ -1,8 +1,9 @@
 import { EDmtPluginType } from '@development-framework/dm-core'
 
 import App from './App'
+import {TPlugin} from "./Types";
```

Comment:
```suggestion
import { TPlugin } from "./Types";
```
(PS: Do you have pre-commit hooks installed? I believe this should be auto-fixed by prettier (?))